java-checks/src/main/java/org/sonar/java/checks/security/HostnameVerifierImplementationCheck.java

```diff
@@ -21,6 +21,7 @@
 
 import com.google.common.collect.ImmutableList;
 import java.util.List;
+import java.util.stream.Collectors;
 import javax.annotation.Nullable;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'javax.annotation.Nullable'. ([squid:UselessImportCheck](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.java%3Ajava&pullRequest=1994&issues=AWK95334cOwGs0mBXWCo&open=AWK95334cOwGs0mBXWCo)

[ISSUE_KEY]: AWK95334cOwGs0mBXWCo